util-cgroups/src/hierarchy.rs

```diff
@@ -414,7 +414,7 @@ mod tests {
         let sysfs_path = PathBuf::from("/some/root/a.slice/me");
         let cgroup = Cgroup::from_fs_path(&h, sysfs_path.clone());
         assert_eq!(cgroup.fs_path(), sysfs_path);
-        assert_eq!(cgroup.canonical_path(), "/a.slice/me");
+        assert_eq!(cgroup.cgroup_path, "/a.slice/me");
```

Comment:
I believe that this comes from a lack of rebase? Existing tests should not be changed.